mockgcp/generated/mockgcp/cloud/alloydb/v1beta/resources.pb.go

```diff
@@ -5316,6 +5307,8 @@ type Instance_ConnectionPoolConfig struct {
 	Enabled bool `protobuf:"varint,12,opt,name=enabled,proto3" json:"enabled,omitempty"`
 	// Optional. Connection Pool flags, as a list of "key": "value" pairs.
 	Flags map[string]string `protobuf:"bytes,13,rep,name=flags,proto3" json:"flags,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
+	// Output only. The number of running poolers per instance.
```

Comment:
This field is added to Instance_ConnectionPoolConfig type, but it is not reflected in instance type (instance_types.go or types.generated.go). Looks like something is missing here?